shub/image/utils.py

```diff
@@ -73,7 +73,13 @@ def get_docker_client(validate=True):
     try:
         import docker
     except ImportError:
-        raise ImportError('You need docker-py installed for the cmd')
+        raise ImportError('You need docker python package installed for the cmd')
+
+    # added for compatiblity with old docker-py
+    if hasattr(docker, 'Client'):
+        docker_client_cls = docker.Client
+    else:
+        docker_client_cls = docker.APIClient
```

Comment:
Can you make the comment a little more detailed? E.g. remove the comment at the top and add `# docker-py (legacy)` and `# docker >= 2.0` into the two branches.